lib/logstash/pluginmanager/install.rb

```diff
@@ -1,82 +1,128 @@
 require 'clamp'
 require 'logstash/namespace'
 require 'logstash/environment'
-require 'logstash/pluginmanager'
 require 'logstash/pluginmanager/util'
-require 'rubygems/dependency_installer'
-require 'rubygems/uninstaller'
 require 'jar-dependencies'
 require 'jar_install_post_install_hook'
 require 'file-dependencies/gem'
 
+require "logstash/gemfile"
+require "logstash/bundler"
+
 class LogStash::PluginManager::Install < Clamp::Command
+  parameter "[PLUGIN] ...", "plugin name(s) or file"
```

Comment:
it is weird that the plugin name is optional. Is this just to allow `bin/plugin install --development` ? If so, I recommend adding a new subcommand `bin/plugin development` or something that does the `--development` behavior in this code.